### PR TITLE
fluff item spawns now

### PR DESCRIPTION
**Pull request by TheDavestDave**
allowed_roles chaplain doesn't let you spawn in an item if you join as chaplain. I have no idea how to fix that

### CODE REVIEW

**Review by Runa-Dacino — CHANGES_REQUESTED** [review]
Change 

`allowed_roles = "Chaplain`

to 

`allowed_roles = list("Chaplain")`

**Review by Very-Soft — DISMISSED** [review]
Fixit, don't just remove the restriction.

**Review by Runa-Dacino — CHANGES_REQUESTED** [review]

**Comment by TheDavestDave** [comment]
> Fixit, don't just remove the restriction.

does it still need a job restriction as it's now a sub varriant of a custome with no amour vaules?

File: code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm [hunk 1]
```diff
@@ -1444,6 +1444,7 @@
 	display_name = "Lucky's amour"
 	ckeywhitelist = list ("thedavestdave")
 	character_name = list("Lucky")
+	allowed_roles = list("Chaplain")
```
Comment (Runa-Dacino) [inline code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm:1447]: This should hopefully fix it, and ensure it stays as a chaplain only thing @Very-Soft

**Review by Runa-Dacino — APPROVED** [review]

**Pull request merged**